tla-io/src/main/scala/at/forsyte/apalache/io/OutputManager.scala

```diff
@@ -48,6 +48,21 @@ object OutputManager extends LazyLogging {
         ProfilingFlag -> false
     )
 
+  // For bug report templates as well as the next iteration of error messages, we will need to reference
+  // lines in the original input. This variable stores them.
+  private var sourceLinesOpt: Option[IndexedSeq[String]] = None
+
+  // Should be called only if input is a .tla file
+  def initSourceLines(file: File): Unit =
+    if (sourceLinesOpt.isEmpty) {
+      val src = Source.fromFile(file)
+      try sourceLinesOpt = Some(src.getLines().toIndexedSeq)
+      finally src.close()
+    }
+
+  def getLineInSrc(n: Int): Option[String] = sourceLinesOpt.map { _(n) }
```

Comment:
I don't understand why this is added in this PR. It doesn't seem to be used anywhere?